nuspec/Cake.Portable.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>cake.portable</id>
+    <title>Cake.Portable (Install)</title>
+    <version>0.0.0</version>
+    <authors>Patrik Svensson</authors>
+    <owners>Patrik Svensson</owners>
+    <summary>Cake (C# Make) is a build automation system with a C# DSL to do things like compiling code, copy files/folders, running unit tests, compress files and build NuGet packages.</summary>
+    <description>The Cake script runner.</description>
+    <projectUrl>http://cakebuild.net/</projectUrl>
+    <packageSourceUrl>https://github.com/cake-build/cake</packageSourceUrl>
+    <projectSourceUrl>https://github.com/cake-build/cake</projectSourceUrl>
+    <docsUrl>http://cakebuild.net/docs</docsUrl>
+    <!--<mailingListUrl></mailingListUrl>-->
+    <bugTrackerUrl>https://github.com/cake-build/cake/issues</bugTrackerUrl>
+    <tags>Cake Script Build</tags>
+    <copyright>Copyright (c) Patrik Svensson, Mattias Karlsson and contributors</copyright>
+    <licenseUrl>https://github.com/cake-build/cake/blob/develop/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://raw.githubusercontent.com/cake-build/graphics/master/png/cake-medium.png</iconUrl>
+    <!--<dependencies>
+      <dependency id="" version="__VERSION__" />
+      <dependency id="" />
+    </dependencies>-->
+    <releaseNotes>https://github.com/cake-build/cake/releases</releaseNotes>
```

Comment:
I will pull in the releasenotes in the same way as we do for the NuGet Packages, and that way they are consistent, and if we change it to something else, we can change it everywhere.